dv/uvm/riscv_dv_extension/ibex_log_to_trace_csv.py

```diff
@@ -33,7 +33,7 @@ def process_ibex_sim_log(ibex_log, csv):
             # Extract instruction information
             m = re.search(r"^\s*(?P<time>\d+)\s+(?P<cycle>\d+)\s+" \
                           "(?P<pc>[0-9a-f]+)\s+(?P<bin>[0-9a-f]+)\s+(?P<instr>.*)" \
-                          ".*x(?P<rd>\d+)=0x(?P<val>[0-9a-f]+)", line)
+                          ".*x(?P<rd>[1-9]\d*)=0x(?P<val>[0-9a-f]+)", line)
```

Comment:
why 1-9 only? what about x20, x30?

Reply:
That's covered, because the '[1-9]' will only apply to the first digit of the register number, the '\d+' will cover any digits after that.